src/js/intl/strings.js

```diff
@@ -400,23 +400,23 @@ exports.strings = {
   ///////////////////////////////////////////////////////////////////////////
   'git-error-branch': {
     '__desc__': 'One of the error messages for git',
-    'en_US': 'You can\'t delete the master branch, the branch you are on, or things that ' +
+    'en_US': 'You can\'t delete the main branch, the branch you are on, or things that ' +
```

Comment:
Ah I must have missed this entire file (but handled the level translation files), now this makes more sense :P